### PR TITLE
Update scrumblr

### PR DESCRIPTION
**Pull request by oiseauroch**
Hello, scrumblr is no more actively maintained. I discussed with colibri team to use their own repository with app debranded.

This version should allow to install scrumblr in a subpath. 
Moreover, scrumblr uses a redis database to store data. Currently, the database is not saved and redis doesn't really allow to save a unique database. This new version spawn a specific redis process for scrumblr so that it can be backup and restored.

### CODE REVIEW

**Comment by oiseauroch** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -15,10 +15,13 @@
         "code": "https://github.com/aliasaria/scrumblr"
```
Comment (ericgaspar) [inline manifest.json]:
```suggestion
        "code": "https://framagit.org/colibris/framemo"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
Thanks for taking care of this app!

Do we need to reinstall redis-server pkg, service and config file knowing that it is installed in YunoHost as default?

**Comment by oiseauroch** [comment]
we do not need to reinstall redis-server pkg but it's just in case redis wouldn't be a yunohost dependency anymore. 

Custom service and config files are necessary to spawn a custom process of redis.

**Comment by ericgaspar** [comment]
Redis is served with the address `127.0.0.1:6379` (this is also set in the framemo `config.js` file. Do we need to define another port (8081) and not use the default one (6379)?

**Comment by oiseauroch** [comment]
Normaly this settings are overwritten in the systemd config files. 
- the `scrumblr_redis.service` file call `redis-server` with custom config file that set specific port 
- the `systemd.service` file overwrite settings from the config file with argument parameters.

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
!testme

**Review by ericgaspar — CHANGES_REQUESTED** [review]
We need to remove the logs in the remove script 

```
ynh_secure_remove --file="/var/log/$app"
```

**Comment by ericgaspar** [comment]
!testme

**Comment by oiseauroch** [comment]
@ericgaspar it seems that the yunohost container has a problem with the redis service. (https://github.com/YunoHost/issues/issues/1962)
Trying to install the old version of scrumblr on a fresh container will success but the application will never be able to reach redis because it doesn't start. 
And then updating from the previous version try to connect to redis and fails.

**Comment by oiseauroch** [comment]
I was wrong in my previous comment. 
!testme

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
@oiseauroch you should have write access to this repo now

**Comment by oiseauroch** [comment]
@ericgaspar Thanks a lot for all advice and support.

**Pull request merged**